Services/WorkspaceService.js

```diff
@@ -0,0 +1,15 @@
+const { WorkspaceRepository } = require('../Repositories');
+
+class WorkspaceService {
+
+	constructor() {
+		this.repository = new WorkspaceRepository();
+	}
+
+	async getTeam(teamId) {
```

Comment:
async getWorkspaceByTeamId(teamId) {
        return await Workspace.findOne({team_id: teamId});
    }

    async createWorkspace(data) {
        let workspace = await this.getWorkspaceByTeamId(data.team_id);
        if (!workspace) {
            workspace = await Workspace.create(data);
        }
    }

Reply:
@HenonoaH can u plz add like above functions to retrieve the multiple workspace data. Bcz it's basic kit if anyone use they will follow with basic implementation. Also u can create callback route to create workspace.